package.json

```diff
@@ -49,6 +50,9 @@
     "plots-smoke": "bash plots/test/smoke.sh",
     "changelog": "conventional-changelog --config ./build/changelog-generator/index.js --infile changelog.md --same-file"
   },
+  "lint-staged": {
+    "*.js": ["eslint"]
```

Comment:
what about
```
"*.js": [
  "eslint --fix"
  "git add"
]
```
so we try to autofix the issues and if it's possible we also add the changed files to the git commit

Reply:
Sure I can add that.